packages/flutter/lib/src/material/colors.dart

```diff
@@ -384,4 +384,47 @@ class Colors {
     800: const Color(0xFF37474F),
     900: const Color(0xFF263238),
   };
+
+  /// All Material design primary color swatches (except grey)
+  static const List<Map<int, Color>> primaries = const <Map<int, Color>>[
+    red,
+    pink,
+    purple,
+    deepPurple,
+    indigo,
+    blue,
+    lightBlue,
+    cyan,
+    teal,
+    green,
+    lightGreen,
+    lime,
+    yellow,
+    amber,
+    orange,
+    deepOrange,
+    brown,
+    // grey intentionally omitted
```

Comment:
Grey seemed like a special case, because it is independent of hue and Colors.grey array contains more levels than the primaries. The particular use case I was solving for was picking a random Material color, and I didn't want something to look disabled, and Colors.primaries.where((c) c != Colors.grey) seemed a bit verbose. But that said, I can see a valid argument for leaving grey in -- it's fine by me either way.